java/src/main/native/CMakeLists.txt

```diff
@@ -58,8 +58,6 @@ endif()
 set(CMAKE_CUDA_FLAGS "${CMAKE_CUDA_FLAGS} -gencode=arch=compute_60,code=sm_60")
 set(CMAKE_CUDA_FLAGS "${CMAKE_CUDA_FLAGS} -gencode=arch=compute_70,code=sm_70 -gencode=arch=compute_70,code=compute_70")
 
-set(CMAKE_CUDA_FLAGS "${CMAKE_CUDA_FLAGS} --expt-extended-lambda --expt-relaxed-constexpr")
-
```

Comment:
Same question regarding removal of `--expt-extended-lambda`.

Reply:
I don't think that this file is used. Also it's sloppy in that it uses `CMAKE_CUDA_FLAGS` unstead of somethign like `CUSPATIAL_JNI_CUDA_FLAGS`.